README.md

```diff
@@ -10,11 +10,95 @@ save your Fiddle either as a GitHub Gist or to a local folder. Once published
 on GitHub, anyone can quickly try your Fiddle out by just entering it in the
 address bar.
 
-<h3 align="center">
-  <a href="https://github.com/electron/fiddle/releases/latest">
-  Download Electron Fiddle for macOS, Windows, and Linux
-  </a>
-</h3>
+# Download
+
+<table class="is-fullwidth">
```

Comment:
Is this CSS class doing anything?

Reply:
According to some obscure internet thingies it might, but I haven't seen it in action just yet